### DISCUSSION TITLE
Automatic Update from Callbook for API Uploads

### DISCUSSION THREAD

**Discussion by DJLax5** [magicbug/Cloudlog#2764] [Ideas / Feature Requests]
Hi everyone,

when I work FT8 or RTTY I upload my QSOs via the CL API. 
The QSOs shown in CL do not have the callbook information right away. If I use the live QSO entry, CL will fetch these pieces of information for me. 
Maybe there could be an option to automatically update the callbook information for empty fields upon API upload? Currently, I do the callbook fetch from the advanced logbook view. 

I am not sure if anybody else would find that useful, but I would like it :)

**Comment by Wireheadbe** [comment]
Hi, 

I just jumped from Log4OM to Cloudlog and am integrating my station via API.
So far I'm sending the Frequency / Mode of my Flex Radio via the API, works superb!
Same for FT8 QSO's as above. 

I run in to the same thing :) Thanks @DJLax5 for the tip about updating the info manually.

I agree, an option to "Use QRZ.com XML to complete QSO info" for QSO's received via API would be handy 👍🏻
